test/tests/calendar/CalendarImporterTest.ts

```diff
@@ -796,6 +796,41 @@ o.spec("CalendarImporterTest", function () {
 				},
 			)
 		})
+		o("recurrence id on event without UID will be deleted", async function () {
```

Comment:
we probably should add one with just recurrence-id

Reply:
not sure what you mean? this test checks that the importer removes recurrence-ids from events where there is no uid. the input has just a recurrence-id, but no uid.